estate/models/estate_property.py

```diff
@@ -0,0 +1,73 @@
+from odoo import api, fields, models
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_compare
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate Property"
+    _order = "id desc"
+    _check_expected_price = models.Constraint("CHECK(expected_price>0)", "Le prix doit être strictement positif.")
+    _check_selling_price = models.Constraint("CHECK(selling_price>=0)", "Le prix doit être positif.")
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(default=fields.Date.add(fields.Date.today(), months=3), copy=False)
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        string='Garden orientation',
+        selection=[('North', 'N'), ('South', 'S'), ('East', 'E'), ('West', 'W')],
+        help="Specify the orientation of the garden to know when you're gonna enjoy the sun")
+    state = fields.Selection(
+        selection=[('New', 'New'), ('Offer Received', 'Offer Received'), ('Offer Accepted', 'Offer Accepted'), ('Sold', 'Sold'), ('Cancelled', 'Cancelled')],
+        default='New'
+    )
+    active = fields.Boolean(default=True)
+    salesman = fields.Many2one("res.users")
+    buyer = fields.Many2one("res.partner", copy=False)
```

Comment:
Should end with _id as they are relational fields. Do not forget to replace them in all your project.
```suggestion
    salesman_id = fields.Many2one("res.users")
    buyer_id = fields.Many2one("res.partner", copy=False)
```